sheets-import/README.md

```diff
@@ -1,102 +1,271 @@
-include in your project. To add it to your script, do the following in the
-Apps Script code editor:
-
-1. Click on the menu item "Resources > Libraries..."
-1. In the "Find a Library" text box, enter the project key
-"MswhXl8fVhTFUH_Q3UOJbXvxhMjh3Sh48" and click the "Select" button.
-1. Choose the latest version in the dropdown box.
-1. Click the "Save" button.
-
-This will add the [OAuth2 library](https://github.com/googlesamples/apps-script-oauth2)
-to your project. Repeat the above steps with the project key
-"MGwgKN2Th03tJ5OdmlzB8KPxhMjh3Sh48" to add Underscore to the project as well.
-
-## Setup: API configuration
-This template requires app-specific configuration before it can used.
-Specifically, the template will need to be informed of the authorization
-details, and certain adjustments made to ensure the correct data can be
-extracted from the responses.
-
-In the Server.gs, APICode.gs and Auth.gs project files, there are several
-comments marked as **TODO**. To configure the template, visit each of these
-**TODO** sections and follow the directions found there.
-
-Note that the template design assumes that the template will connect to an API
-service using OAuth 2.0; however, it would be possible adapt it to connect to
-a different kind of service, such as a SQL database.
-
-## Additional information
-
-For more information, see:
-
-* [Extending Google Sheets](https://developers.google.com/apps-script/guides/sheets)
-* [Known Issues specific to Google Sheets](https://developers.google.com/apps-script/migration/sheets)
-
-Note that this template must be added to a container-bound
-script attached to a Google Sheet in order to function. Developed
-add-ons must go through a
-[publishing process](https://developers.google.com/apps-script/add-ons/publish)
-before they can be made available publicly.
+# Template: Import Data into Google Sheets
+
+An Add-on Template for importing data from a third-party source into
+Google Sheets.
+
+**Features:**
+
+- Rapid integration with third-party data sources like APIs or internal
+  databases,
+- Configurable options for each report;
+  **add text fields, checkboxes, and selectboxes** and reference them in
+  your API call,
+- Scheduled reports that run every 24 hours,
+- Collaborative reports shared between users,
+- Update sheets in the background.
+
+![A demo of Sheets-Import integrated with GitHub](media/demo.gif)
+
+## Quickstart
+
+#### 1. Create the Apps Script Project
+
+1. Navigate to [https://script.google.com](https://script.google.com) and create
+  a new `Blank Project`.
+2. For each file in this repo, create a file of the
+  correct type and paste in the contents.
+  - For an HTML file, select `File` > `New` > `Html File` and enter the
+    full name (i.e. `app.js.html`).
+  - For a Script file, select `File` > `New` > `Script File` and enter the
+    full name (i.e. `server.gs`).
+  - The files you must copy over (descriptions of each at the bottom of the
```

Comment:
These files should have short descriptions here explaining what each file's purpose is.